engine/core/src/main/java/org/datacleaner/descriptors/AnnotationBasedTransformerComponentDescriptor.java

```diff
@@ -62,6 +63,9 @@ public boolean isDistributable() {
         if (annotation != null) {
             return annotation.value();
         }
+        if (ReflectionUtils.is(getComponentClass(), HasDistributionAdvice.class)) {
```

Comment:
This code makes no sense. ReflectionUtils.is(xxx) then it will return true, otherwise it will return true?